libs/dataset-explorer/src/lib/DataExplorerParentTab.tsx

```diff
@@ -0,0 +1,90 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { Pivot, PivotItem, Stack } from "@fluentui/react";
+import { localization } from "@responsible-ai/localization";
+import React from "react";
+
+import { DataBalanceTab } from "./DataBalanceTab";
+import { dataExplorerParentTabStyles } from "./DataExplorerParentTab.styles";
+import { DatasetExplorerTab } from "./DatasetExplorerTab";
+
+interface IDataExplorerParentTabProps {
+  showDataBalanceExperience: boolean;
+}
+
+interface IDataExplorerParentTabState {
+  activeViewOption: DataExplorerParentTabOptions;
+}
+
+enum DataExplorerParentTabOptions {
+  DatasetExplorer = "DatasetExplorer",
+  DataBalance = "DataBalance"
+}
+
+export class DataExplorerParentTab extends React.Component<
```

Comment:
Hmm, I'm not a fan of the naming here. If the overall component remains "Data explorer" then `DataExplorerTab` should be the parent (without further mentioning it in the name). I guess this goes back to the same naming conondrum I mentioned in the other comment 🙁

Reply:
Changed this to `DataAnalysisTab` which contains the Data Explorer and Data Balance tabs! 😊

This is based off Phil's suggestion that we call the overall component "Data analysis" with children "Data explorer" and "Data balance" tabs